packages/obonode/obojobo-chunks-numeric-assessment/NumericAnswer/numeric-option.js

```diff
@@ -157,6 +158,9 @@ const NumericOption = ({ numericChoice, onHandleInputChange, onHandleSelectChang
 				<div className="is-type-range">
 					<label className="select requirement">
 						Answer Type
+						<div style={{ display: 'inline', position: 'relative', left: 0 + 'em', top: 0 + '%' }}>
```

Comment:
Since this same rule is used multiple times throughout the component, either set it as a constant at the top of the file (e.g. `const MORE_INFO_STYLE = {...}`) or set a `className` on the relevant components and define these style rules in an adjacent `.scss` file.

I prefer the second approach.